tensorflow/python/autograph/operators/py_builtins.py

```diff
@@ -427,7 +428,8 @@ def map_(fn, *iterables):
 
 
 def _tf_dataset_map(fn, *iterables):
-  return dataset_ops.DatasetV2.zip(iterables).map(fn)
+  zipped_dataset = dataset_ops.DatasetV2.zip(iterables)
+  return zipped_dataset.map(fn, num_parallel_calls=dataset_ops.AUTOTUNE)
```

Comment:
this is generally not correct because it does not preserve semantics of stateful map functions

please revert this change

Reply:
Thanks for the heads up. I'll send a rollback.

Reply:
Good catch, sorry about that.